packages/server/seeds/dev/01_main.js

```diff
@@ -63,14 +63,18 @@ exports.seed = async (knex) => {
     // populate it after the agency has been created. We are
     // looking at removing main_agency_id entirely.
     // TODO (Brian Spates): Update once main_agency_id is removed
-    await knex.raw(`WITH main_agency_lookup AS (
-                        SELECT id, tenant_id
-                        FROM agencies
-                        WHERE main_agency_id = id
-                    ) UPDATE tenants
-                      SET main_agency_id = mal.id
-                      FROM main_agency_lookup mal
-                      WHERE mal.tenant_id = tenants.id`);
+    //
+    // TODO (pearkes): How should this be modified with the removal of
+    // main_agency_id? It does look like we're using this to set
+    // the tenant.main_agency_id. Should we be reading the parent?
+    // await knex.raw(`WITH main_agency_lookup AS (
```

Comment:
Can this query be removed? I believe this is redundant now but we're setting `tenants.main_agency_id` here with the of the `agencies` lookup. Should we just do a select on `id = id` for agencies here?

Reply:
> Should we just do a select on id = id for agencies here?
I might be missing something here... From what I can see, we still need a step that sets `tenant.main_agency_id` somewhere in the seeding process, so I don't think it can be removed entirely. 

My instinct is that the correct `tenant.main_agency_id` value should be able to be determined by checking for the "root" agency on each tenant, e.g.
```sql
WITH main_agency_lookup AS (
  SELECT id, tenant_id
  FROM agencies
  WHERE parent = 0
) UPDATE tenants 
  SET main_agency_id = mal.id
  FROM main_agency_lookup mal
  WHERE mal.tenant_id = tenants.id;
```
The problem with the above query is that it assumes that there's only one agency per tenant with `parent = 0`, but based on the seed data we're currently using, there are two issues:
1. there are two agencies (one with `name = "Test sub USDR", tenant_id = 1` and another with `name = "B Test sub USDR", tenant_id = 1`) that have the same `tenant_id` value. 
2. It leaves one tenant (with `name = "NPO Tenant"`) without a value for `main_agency_id` because there are no agencies associated with that tenant.

What I'm not sure about is whether the seed data is incorrect (which implies we should have some unique constraint that prevents this situation) or if there's a gap in my understanding of the significance of a "main" agency.

CCing @as1729 to see if his understanding is different than mine. If not, we may want to check with @mhuang-usdr to see if she can clarify the intended business logic here.